tests/bulk_create/tests.py

```diff
@@ -199,6 +200,13 @@ def test_bulk_insert_expressions(self):
         bbb = Restaurant.objects.filter(name="betty's beetroot bar")
         self.assertEqual(bbb.count(), 1)
 
+    def test_nullable_fields(self):
+        # Inserting null mixed with values
+        NullableFields.objects.create(int_or_null=7)
+        NullableFields.objects.bulk_create([
+            NullableFields(int_or_null=i if i % 2 == 0 else None)
```

Comment:
I think the loop makes it look more complicated than it needs to be. How about something like:
```python
NullableFields.objects.bulk_create([
    NullableFields(int_or_null=1),
    NullableFields(int_or_null=None),
])
```